.gitmodules

```diff
@@ -52,3 +52,9 @@
 [submodule "src/ingest-pipeline/submodules/py-hubmapbags"]
 	path = src/ingest-pipeline/submodules/py-hubmapbags
 	url = https://github.com/hubmapconsortium/py-hubmapbags.git
+[submodule "src/ingest-pipeline/airflow/dags/hubmap-inventory"]
+	path = src/ingest-pipeline/airflow/dags/hubmap-inventory
+	url = https://github.com/hubmapconsortium/hubmap-inventory
+[submodule "src/ingest-pipeline/submodules/hubmap-inventory"]
```

Comment:
Does it need to be in two different places?

Reply:
Whhoooops